example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -316,6 +401,16 @@ auto main() -> int
     alpaka::wait(devQueue);
     std::cout << std::endl;
 
+    PrintBufferKernel3D printBufferKernel3D;
+    alpaka::exec<Acc>(
+        devQueue,
+        devWorkDiv,
+        printBufferKernel3D,
+        alpaka::access(deviceBuffer1), // 1st kernel argument
```

Comment:
At the kernel callsite, we create an `accessor` from a buffer using the `access` function. The `access` function returns an accessor, which is a lightweight value type encapsulating all required information for supporting `operator()` later.

We could extend this further with ideas like `alpaka::access<readonly>(buffer)`.